san/sanbase_graphql.py

```diff
@@ -333,11 +333,11 @@ def eth_top_transactions(idx, slug, **kwargs):
 
     query_str = """
     query_{idx}: projectBySlug (slug: \"{slug}\"){{
-            ethTopTransactions (
-                from: \"{from_date}\",
-                to: \"{to_date}\",
-                limit: {limit},
-                transactionType: {transaction_type}
+        ethTopTransactions (
+            from: \"{from_date}\",
+            to: \"{to_date}\",
+            limit: {limit},
+            transaction_type: {transaction_type}
```

Comment:
use camelCase here. It is true that our backend understands both but it's standard (or at least more comment) for the client to use camelCase

Reply:
My mistake, when I was merging the conflicts it got messed up and I deleted it and then thought that its name was with underscore. Fixed it